### PR TITLE
added Armstrong number file

### PR DESCRIPTION
**Pull request by nayanajain**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by nayanajain** [comment]
Added Armstrong_Number.py file

File: Python/math/Armstrong_Number.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# program to check if the number is an Armstrong number or not
```
Comment (atarax665) [inline Python/math/Armstrong_Number.py]: Add an introduction of Armstrong number.

File: Python/math/Armstrong_Number.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# take input from the user
```
Comment (atarax665) [inline Python/math/Armstrong_Number.py]: Don't add such trivial comments, add comments only where needed.

File: Python/math/Armstrong_Number.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+   sum += digit ** 3
+   temp //= 10
+
+# display the result
+if num == sum:
+   print(num,"is an Armstrong number")
+else:
+   print(num,"is not an Armstrong number")
+
+#Output
```
Comment (atarax665) [inline Python/math/Armstrong_Number.py]: Add sample input/output in docstring with Space and time complexity of your program.

File: Python/math/Armstrong_Number.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+#case 1
+#Enter a number: 407
+#407 is an Armstrong number
+#case 2
+#Enter a number: 22
+#22 is not an Armstrong number
+
```
Comment (atarax665) [inline Python/math/Armstrong_Number.py]: Remove these blank lines.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Update appropriate README.md

**Comment by nayanajain** [comment]
..

File: Python/math/Armstrong_Number.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+case 1
+Enter a number: 407
+407 is an Armstrong number
+case 2
+Enter a number: 22
+22 is not an Armstrong number
+
+Time complexity: O(log(n))
+Space complexity: O(1)
+'''
```
Comment (HarshCasper) [inline Python/math/Armstrong_Number.py:30]: Add a new line at the EOF

File: Python/math/Armstrong_Number.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+num = int(input("Enter a number: "))
+sum = 0
+
+# finding  the sum of the cube of each digit
+temp = num
+while temp > 0:
+   digit = temp % 10
+   sum += digit ** 3
+   temp //= 10
```
Comment (HarshCasper) [inline Python/math/Armstrong_Number.py:13]: Push the primary logic inside a modular function

File: Python/math/Armstrong_Number.py [hunk 7]
```diff
@@ -0,0 +1,30 @@
+''' program to check if the number is an Armstrong number or not
+A positive number is called Armstrong Number if it is equal to the sum of cubes of its digits
+'''
+
+num = int(input("Enter a number: "))
+sum = 0
```
Comment (HarshCasper) [inline Python/math/Armstrong_Number.py:6]: Enclose the driver logic in a `if __name__ == '__main__'` block

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
@nayanajain 

Are you still working on the same?

**Comment by nayanajain** [comment]
This program is already present in python...so i stopped working on it

**Pull request closed without merging**